src/50_Samples_%26_Templates/Product.html

```diff
@@ -514,7 +514,7 @@ <h2 class="m1">Apple</h2>
                   layout="container"
                   [selected]="selectedColor"
                   on="select:AMP.setState({selectedColor:event.targetOption, selectedSlide:0,
```

Comment:
One thing I've only just noticed, can you format it like a normal JSON string? This would greatly improve readability.